src/org/labkey/test/tests/component/GridPanelTest.java

```diff
@@ -1856,10 +1863,11 @@ public void testFilterDialogWithViews() throws IOException, CommandException
      *
      * @param exportedFile The exported csv/tsv file.
      * @param expectedValues A list of the expected values in the rows.
-     * @param columns The columns to validate against.
+     * @param columnNames The columns to validate against.
+     * @param columnLabels The columns to validate against.
      * @throws IOException Can be thrown by the file reader.
      */
-    private void validateExportedData(File exportedFile, List<Map<String, Object>> expectedValues, List<String> columns) throws IOException
+    private void validateExportedData(File exportedFile, List<Map<String, Object>> expectedValues, List<String> columnNames, List<String> columnLabels) throws IOException
```

Comment:
Not sure if it's feasible, but consider having this take a list of `FieldInfo` instead of separate lists for names and labels.
```suggestion
    private void validateExportedData(File exportedFile, List<Map<String, Object>> expectedValues, List<FieldInfo> columns) throws IOException
```

Reply:
Done